src/test/java/com/hivemq/cli/commands/hivemq/datapolicy/DataPolicyCreateCommandTest.java

```diff
@@ -41,9 +41,9 @@
 
 public class DataPolicyCreateCommandTest {
 
-    private final @NotNull HiveMQRestService hiveMQRestService = mock(HiveMQRestService.class);
+    private final @NotNull HiveMQRestService hiveMQRestService = mock();
     private final @NotNull Gson gson = new Gson();
-    private final @NotNull OutputFormatter outputFormatter = mock(OutputFormatter.class);
+    private final @NotNull OutputFormatter outputFormatter = mock();
     private final @NotNull DataHubDataPoliciesApi policiesApi = mock(DataHubDataPoliciesApi.class);
```

Comment:
```suggestion
    private final @NotNull DataHubDataPoliciesApi policiesApi = mock();
```

Also in other classes